lua/cfc_notifications/shared/sh_presets.lua

```diff
@@ -98,60 +126,20 @@ CFCNotifications.registerNotificationType( "Buttons", function( CONTEXT )
             end
 
             btn:SetSize( btnW, btnH )
-            btn:SetPos( 10 + ( k - 1 ) * btnTotalW, btnY )
+            btn:SetPos( 10 + ( btnCol - 1 ) * btnTotalW, btnY )
             table.insert( btns, btn )
-        end
-
-        self._btns = btns
-    end
-end )
 
-CFCNotifications.registerNotificationType( "TextAcknowledge", function( CONTEXT )
```

Comment:
Why are we removing this type? It adds a type for a notif that fades away to not be obstructive but still require user input. Have these been replaced by something adverts?

Reply:
Oop, that was an accident